ckan/requirements.txt

```
@@ -17,11 +17,11 @@ chardet==3.0.4
 -e git+https://github.com/GSA/ckanext-datajson.git@a5d8c9458a7efe955c31b90eeac1e7797881d014#egg=ckanext-datajson
 -e git+https://github.com/ckan/ckanext-dcat@c4e7319aff55e5315831b28e918299092f2d5805#egg=ckanext-dcat
 ckanext-envvars==0.0.1
--e git+https://github.com/GSA/ckanext-geodatagov.git@bc700f671cd1d890db265c65f5db1cba121297f4#egg=ckanext-geodatagov
+-e git+https://github.com/GSA/ckanext-geodatagov.git@ded11ffd3e4c97b8d418e45bfeeea0c3f4f10796#egg=ckanext-geodatagov
 -e git+https://github.com/GSA/ckanext-googleanalyticsbasic.git@c6a425d5e14d658c0fa3661fdc4423162161c3f4#egg=ckanext-googleanalyticsbasic
--e git+https://github.com/ckan/ckanext-harvest.git@d91ea66cf083b6618c5dab7ba56f6d7a8b468378#egg=ckanext-harvest
+-e git+https://github.com/ckan/ckanext-harvest.git@63701adaba22af6c82f699aa0d9c045ced9be4a3#egg=ckanext-harvest
 -e git+https://github.com/ckan/ckanext-qa.git@d7d384cb18a85243ea62ef0bcc76bc1775f1c806#egg=ckanext-qa
--e git+https://github.com/davidread/ckanext-report.git@2abba3fa2ddf29f4decc4a0c4c39101fb994db23#egg=ckanext-report
+-e git+https://github.com/davidread/ckanext-report.git@82af48608b99e7176f1d76f70e59fcfdadf11dfd#egg=ckanext-report
```

```suggestion
-e git+https://github.com/ckan/ckanext-report.git@82af48608b99e7176f1d76f70e59fcfdadf11dfd#egg=ckanext-report
```

`davidread` redirects to `ckan` so I figured we should update it.  Another note, this includes a bunch of py3 upgrades, the tests pass for CKAN 2.8 and PY2, so this should be enough, just noting here in case bugs creep up, https://github.com/ckan/ckanext-report/runs/3684030081

---

good catch. i updated the repo link